docs/source/reference/links.rst

```diff
@@ -191,6 +191,13 @@ where ``fc7`` denotes a layer before the last fully-connected layer.
 Unlike the usual links, these classes automatically load all the
 parameters from the pre-trained models during initialization.
 
+GoogLeNet
+~~~~~~~~~
+.. autoclass:: GoogLeNet
+   :members:
+
+.. autofunction:: chainer.links.model.vision.googlenet.prepare
+
```

Comment:
I moved GoogLeNet next to VGG.
(Since the original order of VGG and ResNet can be seen as from weak model to powerful one for me, so I put GoogLeNet first as the weakest model.)